sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -1,103 +1,151 @@
 <template>
-  <modal-dialog-base
-    title="Application already in progress"
-    dialogType="warning"
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-  >
-    <template v-slot:content>
-      <v-container>
-        <form>
-          <p>There is already a draft of an application in progress.</p>
-          <p>Please continue your draft application or cancel it.</p>
-        </form>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <footer-buttons
-        secondaryLabel="Close"
-        :showPrimaryButton="false"
-        @secondaryClick="dialogClosed"
-      />
-    </template>
-  </modal-dialog-base>
-  <student-page-container>
-    <v-sheet elevation="1" class="mx-auto">
-      <v-container>
-        <formio
-          formName="studentapplication"
-          :data="initialData"
-          @loaded="formLoaded"
-          @customEvent="customEventCallback"
-        ></formio>
-      </v-container>
-    </v-sheet>
-  </student-page-container>
+  <v-form ref="startApplicationForm">
+    <modal-dialog-base
+      title="Application already in progress"
+      dialogType="warning"
+      :showDialog="showDialog"
+      @dialogClosed="closeDialog"
+    >
+      <template v-slot:content>
+        <v-container>
+          <form>
+            <p>
+              There is already an application selected with this program year.
+              Please continue with that application or cancel it to start a new
+              application.
+            </p>
+          </form>
+        </v-container>
+      </template>
+      <template v-slot:footer>
+        <footer-buttons
+          primaryLabel="Close"
+          :showSecondaryButton="false"
+          @primaryClick="closeDialog"
+        />
+      </template>
+    </modal-dialog-base>
+
+    <student-page-container container-class="container-max-width ma-auto">
+      <template #header>
+        <header-navigator
+          title="Applications"
+          subTitle="Start New Application"
+          :routeLocation="{
+            name: StudentRoutesConst.STUDENT_APPLICATION_SUMMARY,
+          }"
+        >
+          <template #buttons>
+            <start-application />
+          </template>
+        </header-navigator>
+      </template>
+      <h2 class="category-header-large primary-color pb-4">
+        Apply for funding
+      </h2>
+      <content-group>
+        <div class="category-header-medium-small pa-2">
+          Financial Aid Application
+        </div>
+        <div class="pb-6 px-2">
+          Apply to see if you are applicable for provincial or federal loans and
+          grant. This is for students attending full-time or part-time
+          post-secondary studies.
+        </div>
+        <div class="category-header-medium-small pb-3 px-2">
+          Select program year
```

Comment:
@hellolynn-tbtb there is a small catch here. The "Select program year" is not a label that points to the input. I guess that this is not causing issues with accessibility but it can be double-checked.
That said, in the forms that we adopted 'vuetifty" we haven`t been adding the label on top (Vuetify has a different style), but for students, I guess that this is the first form input using Vuetify (if I am not wrong). Based on this the question is: should we try to adapt the forms to look like form.io (label on top)?

![image](https://user-images.githubusercontent.com/61259237/199789477-a8324100-8280-40d3-9d40-3cb915172334.png)

Reply:
Talked with @hellolynn-tbtb about it, please move the "Select program year" `div` and make it the `v-select` label.